pulp_ansible/app/migrations/0004_add_fulltext_search_indexes.py

```diff
@@ -18,7 +18,7 @@
           coalesce(string_agg("ansible_tag"."name", ' '), '')
         ), 'B')
       FROM
-        "ansible_tag" INNER JOIN "ansible_collectionversion_tags" ON ("ansible_tag"."_id" = "ansible_collectionversion_tags"."tag_id")
+        "ansible_tag" INNER JOIN "ansible_collectionversion_tags" ON ("ansible_tag"."pulp_id" = "ansible_collectionversion_tags"."tag_id")
```

Comment:
@daviddavis I didn't know how to solve it without changing this migration

Reply:
I think we probably have to drop and recreate the vector. cc @bmbouter

Reply:
how can I drop it?

Reply:
Given this is still pre-GA, and the complexity of having to "fix" the migration with hand-generated code in a new migration I'm ok with modifying this migration in-place.

Reply:
Either squashing or updating the migration is cool with me.